AISKU/scripts/setActiveCdnVersion.ps1

```diff
@@ -617,31 +644,42 @@ Function SetActiveVersion(
     foreach ($theBlob in $fileList) {
         $blob = $theBlob.blob
         $blobContext = $theBlob.context.azureContext
-        Log $("Copying: {0,-40} {1,6:N1} Kb  {2:yyyy-MM-dd HH:mm:ss}" -f $($blob.ICloudBlob.Container.Name + "/" + $blob.Name),($blob.Length/1kb),$blob.LastModified)
+        Log $("Copying: {0,-55} {1,6:N1} Kb  {2:yyyy-MM-dd HH:mm:ss}" -f $($blob.ICloudBlob.Container.Name + "/" + $blob.Name),($blob.Length/1kb),$blob.LastModified)
 
         $version = GetVersion $blob.Name
         if ($null -ne $version) {
-            $verParts = $version.ver.Split(".")
+            $verDetails = Get-VersionDetails $version.ver
+            $verParts = $verDetails.ver.Split(".")
             if ($verParts.Length -ne 3) {
-                Log-Failure "ScriptError: Invalid Version! [$activeVersion]"
+                Write-LogFailure "ScriptError: Invalid Version! [$activeVersion]"
+            }
+
+            $preRel = ""
+            if ($verDetails.type -ne "release") {
+                $preRel = "-" + $verDetails.type
+
+                if ($verDetails.type -eq $container) {
+                    # remove any "beta" tag when deploying to the beta container etc.
```

Comment:
This change means that when accessing ```js.monitor.azure.com/beta/ai.2.min.js``` would actually return the beta release, but if deployed to ```next``` then ```js.monitor.azure.com/next/ai.2.min.js``` would still be the official release and the beta would need to use ```js.monitor.azure.com/next/ai.2-beta.min.js```